lib/getStepDefinitionsPaths.js

```diff
@@ -1,23 +1,35 @@
 const glob = require("glob");
 const cosmiconfig = require("cosmiconfig");
+const process = require("process");
 const stepDefinitionPath = require("./stepDefinitionPath.js");
 const { getStepDefinitionPathsFrom } = require("./getStepDefinitionPathsFrom");
 
 const getStepDefinitionsPaths = filePath => {
   let paths = [];
+  const featureBase = stepDefinitionPath();
+  const appRoot = process.cwd();
   const explorer = cosmiconfig("cypress-cucumber-preprocessor", { sync: true });
   const loaded = explorer.load();
   if (loaded && loaded.config && loaded.config.nonGlobalStepDefinitions) {
-    const nonGlobalPattern = `${getStepDefinitionPathsFrom(
-      filePath
-    )}/**/*.+(js|ts)`;
-    const commonPath =
-      loaded.config.commonPath || `${stepDefinitionPath()}/common/`;
+    let nonGlobalPath = getStepDefinitionPathsFrom(filePath);
+    let commonPath = loaded.config.commonPath || `${featureBase}/common/`;
```

Comment:
@liamchilds  @lgandecki 
I have tried this PR to test if it resolves the issue described in https://github.com/TheBrainFamily/cypress-cucumber-preprocessor/pull/275 (step definitions defined in a custom commonPath are not found in my setup (webpack, typescript)).

It did not because the `appRoot` is only used when `nonGlobalStepBaseDir` is used. When changing this line to
```typescript
let commonPath = `${appRoot}/${loaded.config.commonPath}` || `${featureBase}/common/`;
```
the `commonPath` parameter works for me, even without specifying `nonGlobalStepBaseDir`.

Reply:
thanks for the explanation, I think I will take this PR and add the appRoot as you suggested. Just have to test it a bit.